ape_ganache/provider.py

```diff
@@ -51,6 +60,11 @@ class GanacheNetworkConfig(PluginConfig):
     # Used only in GanacheForkProvider.
     fork: Dict[str, Dict[str, GanacheForkConfig]] = {}
 
+    wallet: GanacheWalletConfig = GanacheWalletConfig()
+    # For setting the values in --wallet.* command arguments
+    # BESIDES the mnemonic, HD Path, and number of accounts. Use
+    # the ``test`` config to set those.
```

Comment:
Nitpick: maybe different phrasing?

```suggestion
    # wallet allows setting values in --wallet.* command arguments
    # Use the ``test`` config to set the mnemonic, HD Path, and number of accounts.
```